src/linagora.esn.unifiedinbox/app/controllers.js

```diff
@@ -481,6 +484,14 @@ angular.module('linagora.esn.unifiedinbox')
                 self.fixTime('toDate');
                 self.updateDateAndTime('toDate');
               }
+
+              Object.defineProperty(self.vacationDate, 'start', {
+                get() { return esnDatetimeService.format($scope.vacation.fromDate.toDate(), 'LLL') },
```

Comment:
Why did you take `LLL`? Is it equal to `esnDatetime:'mediumDate time'`? Why not use mediumDate constant from esnDatetimeService?

Reply:
I use $filter for it. 'mediumDate + time' --> 'longDate'